spec/models/user_spec.rb

```diff
@@ -41,4 +41,12 @@
       end
     end
   end
+
+  describe "with a user with an MfaConfig" do
+    let!(:user) { create(:authy_user) }
```

Comment:
eventually, it might be worthwhile to rename to `verify_user` eventually, but I think that's for another PR